addons/website_sale/static/src/scss/website_sale.scss

```
@@ -318,45 +301,99 @@ $input-border-color: $gray-400;
             }
         }
 
-        // Sliglty increase padding for larger blocks
-        @for $x from 2 through 4 {
-            [class*="o_wsale_product_grid_wrapper_#{$x}_"] {
-                --o-wsale-card-info-padding: #{map-get($spacers, 3)} 0 #{map-get($spacers, 2)};
+        // Stretch 
+        .oe_product.oe_product_size_stretch {
+            --o-wsale-card-border-radius: 0;
+            --o-wsale-card-flex-direction: row;
+            --o-wsale-card-info-padding: 0 #{map-get($spacers, 3)};
+            --o-wsale-card-thumb-size: calc(calc(100cqw / var(--o-wsale-ppr)) - calc(var(--o-wsale-products-grid-gap) / 2));
+            --o-wsale-card-padding: #{0 0 $o-wsale-products-layout-grid-gutter-width};
+
+            .oe_product_image, .oe_product_image_link {
+                height: 100%;
+                padding: 0;
+            }
+
+            .oe_subdescription div {
+                -webkit-line-clamp: initial;
+                line-height: inherit;
+            }
+        }
+
+        &:not(.o_wsale_context_thumb_cover) .oe_product.oe_product_size_stretch {
+            .oe_product_image img {
+                object-position: top left;
             }
+        }
 
-            .o_wsale_design_cards [class*="o_wsale_product_grid_wrapper_#{$x}_"] {
-                --o-wsale-card-info-padding: #{map-get($spacers, 3)} #{map-get($spacers, 3)} #{map-get($spacers, 2)};
+        &.o_wsale_design_cards .oe_product.oe_product_size_stretch {
+            --o-wsale-card-padding: 0;
+            --o-wsale-card-border-radius: #{$card-border-radius};
+            --o-wsale-card-info-padding: #{map-get($spacers, 3)};
+        }
+
+        &.o_wsale_design_grid .oe_product.oe_product_size_stretch {
+            --o-wsale-card-padding: 0;
+            --o-wsale-card-info-padding: #{map-get($spacers, 2)} #{map-get($spacers, 3)};
+
+            &[data-colspan="4"] {
+                --o-wsale-card-thumb-size: calc(calc(100cqw / 2));
+            }
+        }
+
+        &.o_wsale_context_thumb_cover .oe_product.oe_product_custom_portrait {
+            .oe_product_image, .oe_product_image_link {
+                height: 100%;
+                padding: 0;
             }
         }
     }
-}
 
-#products_grid.o_wsale_layout_list {
-    @include wsale-break-table($-list: true);
+    .o_wsale_product_information {
+        container-type: inline-size;
+        container-name: o-wsale-grid-product-information;
 
-    .oe_product {
-        --o-wsale-card-border-radius: 0;
-        --o-wsale-card-flex-direction: row;
-        --o-wsale-card-info-padding: 0 #{map-get($spacers, 3)};
+        .o_wsale_products_item_title {
+            font-size: unquote("clamp(0.95rem, 0.5rem + 3cqw, 1.4rem)");
```

What's the purpose of `unquote()` here ? 🤔 
Can't we simply define a clamp like this ? 
```suggestion
            font-size: clamp(0.95rem, 0.5rem + 3cqw, 1.4rem);
```

Same below

---

Nope, SASS doesn't like it 😬 → `Error: Incompatible units: 'cqw' and 'rem'.`